public/src/css/includes/index.less

```diff
@@ -6,6 +6,10 @@
 
 .model-actions, .options {
 
+  .form-control {
```

Comment:
All these changes are 2 spaced, should they be 4? It's up to you but I prefer 2 spaced without padding for css/less/scss etc anyway.

Reply:
The whole file is 2 spaced, so I kept it consistent.